SimulationResult/schema.json

```diff
@@ -53,58 +53,10 @@
                     ],
```

Comment:
you do not have to remove outputParameters. Every user will chose their option

Reply:
Thanks for this edit - I have addressed this in the new commit.